package.json

```diff
@@ -38,6 +38,7 @@
     "body-parser": "1.17.2",
     "bounding-client-rect": "1.0.5",
     "browser-filesaver": "1.1.0",
+    "bundle-loader": "0.5.5",
```

Comment:
We should [regenerate `npm-shrinkwrap.json`](https://github.com/Automattic/wp-calypso/blob/master/docs/shrinkwrap.md) since `package.json` has been updated.